fossa/info/action.yml

```diff
@@ -67,3 +79,38 @@ runs:
         echo "head-revision=${HEAD_REVISION}"
       } >> $GITHUB_OUTPUT
     shell: bash
+  - name: Get additional PR info
+    if: ${{ steps.info.outputs.is-pull-request == 'true' }}
+    id: info-pr
+    env:
+      BASE_REF: ${{ steps.info.outputs.base-ref }}
+      BASE_REVISION: ${{ steps.info.outputs.base-revision }}
+      GH_TOKEN: ${{ inputs.github-token }}
+    run: |
+      # Check recent commits of the base branch for existing scanning results
+      # history depth to search for scanning results
+      depth=5
```

Comment:
maybe a bit overkill but: Could it make sense to make this configurable as another input which defaults to 5?